vmdb/app/models/mixins/ems_openstack_mixin.rb

```diff
@@ -27,7 +27,9 @@ def openstack_handle(options = {})
       raise "no credentials defined" if self.missing_credentials?(options[:auth_type])
 
       username = options[:user] || self.authentication_userid(options[:auth_type])
-      password = options[:pass] || self.authentication_password(options[:auth_type])
+
+      # the |"" is here to prevent an unhelpful ArgumentError from OpenStackHandle::Handle#connect
+      password = options[:pass] || self.authentication_password(options[:auth_type]) || ""
```

Comment:
OK... so.. I'll remove the `|| ""` here, and add those methods so we can try it out...